src/MovieDetails/MovieDetails.js

```diff
@@ -1,24 +1,72 @@
 import React from 'react';
+import ReactPlayer from 'react-player';
 import './MovieDetails.css';
 
-const MovieDetails = ({ movie, statusCode, error }) => {
+const MovieDetails = ({ movie, statusCode, error, movieTrailer }) => {
+
+	const formatNum = (num, type) => {
+		return (
+      num > 0 && 
+      <section className='details-box'>
+        <p className='l-details'><b> {type} :</b></p>
+        <p className='r-details'>${new Intl.NumberFormat('en-US').format(num)}</p>
+      </section>
+		)
+  }
+
+  const formatGenres = (movie) => {
+    if (!movie.genres) {
+      return "Unavailable"
+    } else if (movie.genres.length > 1) {
+      let commaList = movie.genres.reduce((genres, genre) => {
+        genres += ' | ' + genre
+        return genres
+      }, '')
+      return commaList.slice(3, commaList.length);
+    } else {
+      return movie.genres[0];
+    }
+  }
+
   return (
     <section>
       {statusCode < 400 ? 
-				<section>
-					<img className='card-img' src={movie.backdrop_path} />
-					<h1>{movie.title}</h1>
-					<p>{movie.overview}</p>
-					<p>Average User Rating: {movie.average_rating}</p>
-					<p>Genre: {movie.genres[0]}</p>
-					<p>Release Date: {movie.release_date}</p>
-					<p>Budget: ${movie.budget}</p>
-					<p>Total Revenue: ${movie.revenue}</p>
-					<p>Runtime: {movie.runtime} minutes</p>
+				<section className='movie-details-box'>
+          <ReactPlayer url={`https://www.youtube.com/watch?v=${movieTrailer.key}`} />
+					<img className='card-img backdrop' src={movie.backdrop_path} />
+          <section className='up-first'>
+            <img className='card-img poster' src={movie.poster_path} />
+            <section className='poster-aside'>
+              <h1 className='movie-title'><b>{movie.title}</b></h1>
+              <section className='ratings-box'>
+                <p className='l-details'><b>Average User Rating:</b></p>
+                <p className='r-details'>{movie.average_rating.toFixed(2)}</p>
+              </section>
+            </section>
+          </section>
+
+					<section className='overview-box'>
+            <p className='overview-title'><b>Movie Info:</b></p>
+            <p className='overview-text'>{movie.overview}</p>
+          </section>
+					<section className='details-box'>
+            <p className='l-details'><b>Genre:</b></p>
```

Comment:
Maybe we can create a component for movie details-box since we're using it multiple times